solver/scheduler.go

```diff
@@ -368,7 +399,7 @@ func (pf *pipeFactory) NewInputRequest(ee Edge, req *edgeRequest) pipe.Receiver
 	target := pf.s.ef.getEdge(ee)
 	if target == nil {
 		return pf.NewFuncRequest(func(_ context.Context) (interface{}, error) {
-			return nil, errors.Errorf("failed to get edge: inconsistent graph state")
+			return nil, errors.Errorf("failed to get edge: inconsistent graph state in edge %s %s %d", ee.Vertex.Name(), ee.Vertex.Digest(), ee.Index)
```

Comment:
We should keep the error the same, but print more information using `bklog`, like in the other places.

Reply:
Definitely more useful, so not a blocker - but if this was logged, we can format by field, instead of having just a single unstructured line.